casadm/extended_err_msg.c

```diff
@@ -231,8 +231,7 @@ struct {
 	},
 	{
 		KCAS_ERR_CONTAINS_PART,
-		"Device contains partitions.\nIf you want to continue, "
-		"please use --force option.\nWarning: all data will be lost!"
+		"Operation not allowed. Device contains partitions."
```

Comment:
This breaks message printed in scenarios other than `--standby --activate`. We want to change message only in this specific scenario, keeping it as it was for other ones.

Reply:
Ok, custom message will be printed by casadm for this case